### PR TITLE
Swapped NLog with Serilog

### PR DESCRIPTION
**Pull request by AngeloTadeucci**
- Implemented Serilog and removed NLog;
- Fixed all logging templates, using string interpolation is **not** recommended.

### CODE REVIEW

**Review by AlanMorel — APPROVED** [review]

**Review by darkvergus — APPROVED** [review]

File: MapleServer2/MapleServer.cs [hunk 1]
```diff
@@ -25,13 +26,27 @@ public static class MapleServer
...
-    private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
+    private static ILogger _logger;
 
     public static async Task Main()
     {
+        // Setup Serilog
+        Log.Logger = new LoggerConfiguration()
+            .MinimumLevel.Debug()
+            .WriteTo.Console(new ExpressionTemplate(
+                "[{@t:HH:mm:ss}] [{@l:u3}]" +
```
Comment (Sparkymod) [inline MapleServer2/MapleServer.cs:37]: you can store this as a `string expressionTemplate`

File: MapleServer2/MapleServer.cs [hunk 2]
```diff
@@ -25,13 +26,27 @@ public static class MapleServer
...
+                "{#if SourceContext is not null} {Substring(SourceContext, LastIndexOf(SourceContext, '.') + 1),-15}:{#end} {@m}\n{@x}",
+                theme: TemplateTheme.Literate))
+            .WriteTo.File($"{Paths.SOLUTION_DIR}/Logs/MapleServer2/LOG-.txt",
+                rollingInterval: RollingInterval.Day,
+                outputTemplate: "[{Timestamp:yyyy-MM-dd HH:mm:ss}] [{Level}] {SourceContext:l}: {Message:lj}{NewLine}{Exception}")
```
Comment (Sparkymod) [inline MapleServer2/MapleServer.cs:42]: this one too as `string outputTemplate`

File: MapleWebServer/Program.cs [hunk 3]
```diff
@@ -10,8 +13,24 @@
 
 DotEnv.Load(dotenv);
 
+Log.Logger = new LoggerConfiguration()
+    .MinimumLevel.Debug()
+    .WriteTo.Console(new ExpressionTemplate(
+        "[{@t:HH:mm:ss}] [{@l:u3}]" +
```
Comment (Sparkymod) [inline MapleWebServer/Program.cs:19]: here too

**Review by Sparkymod — APPROVED** [review]

**Comment by AngeloTadeucci** [comment]
@Sparkymod done

**Review by darkvergus — APPROVED** [review]

**Pull request merged**